crates/ty_python_semantic/src/semantic_index/use_def.rs

```diff
@@ -192,6 +192,17 @@
 //! for that place that we need for that use or definition. When we reach the end of the scope, it
 //! records the state for each place as the public definitions of that place.
 //!
+//! ```python
+//! x = 1
+//! x = 2
+//! y = x
+//! if flag:
+//!     x = 3
+//! else:
+//!     x = 4
+//! z = x
+//! ```
+//!
```

Comment:
I copied this example here again from above, since there was a different code snippet in between, which was not what the text below intends to refer to.